tests/git.zunit

```diff
@@ -4699,6 +4699,8 @@
     echo >> directory2/$'file4\ncontaining\nnewlines'
     echo >> ' file5 containing space '
     echo >> directory2/$'file6\ncontaining\nnewlines'
+    touch file_outside_tree_object
```

Comment:
In order for the testing framework to detect what's happening, add `run` for every execution like this.
```suggestion
    run touch file_outside_tree_object
```